lib/option.fz

```diff
@@ -163,18 +163,18 @@ is
 
   # return function
   #
-  public fixed type.return (a T) => option a
+  public fixed type.return (a T) => from a
 
 
 
-# option with 1 argument provides an short-hand to wrap a value into an option
-#
-# Using this enables to write
-#
-#   o := option x
-#
-# insted of
-#
-#   o option TypeOfX := x
-#
-public option(T type, o option T) => o
```

Comment:
please do not remove this, `type.from` does not permit type inference  and is more typing even if type inference is not used.